docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -79,16 +86,15 @@ $ ulimit -n <value>
 -u (processes/threads): 64000
 ```
 
-{{< note title="Note" >}}
+{{< note title="Restart servers" >}}
 
-- After changing a ulimit setting, the YB-Master and YB-TServer servers must be restarted in order for the new settings to take effect. Check the `/proc/<process pid>` file to see the current settings.
-- Changes made using ulimit may revert following a system restart depending on the system configuration.
+After changing a ulimit setting, the YB-Master and YB-TServer servers must be restarted in order for the new settings to take effect. Check the `/proc/<process pid>` file to see the current settings.
```

Comment:
We can mention here that ulimits are printed in yb-tserver.INFO file when it's started. 

That is the place to look to know for 100% certain that the limits are applied.